src/Agent.Plugins/PipelineCache/PipelineCacheTaskPluginBase.cs

```diff
@@ -15,12 +15,10 @@ public abstract class PipelineCacheTaskPluginBase : IAgentTaskPlugin
     {
         protected const string RestoreStepRanVariableName = "RESTORE_STEP_RAN";
         protected const string RestoreStepRanVariableValue = "true";
-
         private const string SaltVariableName = "AZDEVOPS_PIPELINECACHE_SALT";
```

Comment:
we should update this variable name as well to match the naming pattern of the other ... so `AZP_CACHING_SALT`